static/js/lib/validation.js

```diff
@@ -89,6 +89,14 @@ export const validateChannelEditForm = validate([
   )
 ])
 
+export const validateChannelBasicEditForm = validate([
+  validation(
+    R.isEmpty,
+    R.lensPath(["value", "link_type"]),
+    "One of the post type options must be selected"
```

Comment:
Maybe `"At least one"` ?

Reply:
also it's super trivial, but it would probably be good to at least have a simple regression unit test for this